spec/forms/decidim/participatory_documents/admin/document_form_spec.rb

```diff
@@ -59,6 +61,22 @@ module Decidim::ParticipatoryDocuments::Admin
           expect(subject.box_opacity).to be_nil
         end
       end
+
+      context "when the document has no file" do
```

Comment:
Are we missing a the case of uploading an invalid file?

Reply:
this is done